libraries/libfc/test/crypto/CMakeLists.txt

```diff
@@ -22,7 +22,7 @@ target_link_libraries( test_k1_recover fc )
 add_test(NAME test_cypher_suites COMMAND test_cypher_suites WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
 add_test(NAME test_webauthn COMMAND test_webauthn WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
 add_test(NAME test_hash_functions COMMAND test_hash_functions WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
-add_test(NAME test_alt_bn128 COMMAND test_alt_bn128 WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
+# add_test(NAME test_alt_bn128 COMMAND test_alt_bn128 WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
```

Comment:
a few things commented out here, not sure if they need to be restored, removed, or fixed

Reply:
The test is using libff APIs, so it needs to be removed. The bn256 submodule have the corresponding tests already.